packages/odyssey-react-mui/package.json

```diff
@@ -66,11 +73,12 @@
     "react-dom": ">=17 <19"
   },
   "scripts": {
+    "build:i18n": "node scripts/properties-to-json.js bundle",
     "build:clean": "rm -rf dist",
     "build:icons": "svgr ../odyssey-icons/src --out-dir ./src/iconDictionary",
     "build:source": "NODE_ENV=production BROWSERSLIST_ENV=modern babel src --out-dir dist --extensions .js,.jsx,.ts,.tsx --source-maps",
     "build:types": "tsc --project tsconfig.production.json",
-    "build": "npm-run-all --sequential build:clean --parallel build:types build:source",
+    "build": "npm-run-all --sequential build:clean build:i18n --parallel build:types build:source",
```

Comment:
I believe `build:i18n` needs to be ran before `build:source` as the `ts` versions need to be available first